business/definitionService.js

```diff
@@ -70,6 +70,7 @@ class DefinitionService {
     const existing = await this._cacheExistingAside(coordinates, force)
     let result
     if (get(existing, '_meta.schemaVersion') === currentSchema) {
+      // Log line used for /status page insights
```

Comment:
Excellent adding the comments about which log lines are used on the `/status` page.  That is handy information to have.